### PR TITLE
STONEBLD-1207: Use buildah build

### PR DESCRIPTION
**Pull request by Michkov**
buildah bud could be confusing, alias build is more clear.

### CODE REVIEW

File: task/update-infra-deployments/0.1/README.md [hunk 1]
```diff
@@ -16,3 +16,7 @@ Clones redhat-appstudio/infra-deployments repository, runs script in 'SCRIPT' pa
 |GITHUB_APP_ID|ID of Github app used for updating PR|305606|false|
 |GITHUB_APP_INSTALLATION_ID|Installation ID of Github app in the organization|35269675|false|
 
+## Workspaces
```
Comment (MartinBasti) [inline task/update-infra-deployments/0.1/README.md:19]: is this change related?

**Review by psturc — APPROVED** [review]
